### ISSUE TITLE
Create rule S6821 (`jsx-a11y/aria-role`): DOM elements with ARIA roles should have a valid non-abstract role

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4238]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/aria-role.md#jsx-a11yaria-role)

https://github.com/SonarSource/rspec/pull/3268
